Content.Server/StationEvents/Components/PhoneBillRuleComponent.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Content.Server.StationEvents.Events;
+using Robust.Shared.Audio;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent]
+public sealed partial class PhoneBillRuleComponent : Component
+{
+    /// <summary>
+    ///     How long until payment is collected.
+    /// </summary>
+    [DataField]
+    public TimeSpan Delay = TimeSpan.Zero;
+
+    /// <summary>
+    ///     Cost of the phone bill per PDA or ID.
+    /// </summary>
+    [DataField]
+    public int Price = 0;
+
+    /// <summary>
+    ///     A list of PDAs and IDs on someone, keyed by that person.
+    /// </summary>
+    [DataField(readOnly:true)]
```

Comment:
```suggestion
    [DataField]
```
I believe `readOnly` is actually the opposite of what you want here - it prevents changing the value after it has been set in the YAML.